scripts/utils/migrate/fetch-redirects.js

```diff
@@ -0,0 +1,35 @@
+const fetch = require('node-fetch');
+const logger = require('../logger');
+const { paginate } = require('./docs-api');
+
+require('dotenv').config();
+
+const fetchRedirects = async (nodeId) => {
```

Comment:
It doesn't look like this is used. Is this needed?

Reply:
that was the method i used for first approach, but ditched it in favor of getting all at once. i left it in case we needed a way to get redirects by passing the docsId to it. i'll remove for now to clean up the file some